lib/appExtensions.ts

```diff
@@ -0,0 +1,221 @@
+// Creates a new App Extension on installation of Sample App
+export async function createAppExtension(
+  storeHash: string,
+  accessToken: string
+) {
+  const response = await fetch(
+    `https://api.bigcommerce.com/stores/${storeHash}/graphql`,
```

Comment:
I probably wouldn't hide the api call inside of the lib folder here - I think this might actually be better as part of the API - we do something similar with checkout: https://github.com/bigcommerce/sample-app-nodejs/blob/automated-billing-beta/pages/api/checkout/%5Bpid%5D.ts#L14

Reply:
Removed api calls from appExtensions file